lib/SymMessageParser/index.js

```diff
@@ -2,14 +2,13 @@ var SymBotAuth = require('../SymBotAuth')
 
 var SymMessageParser = {}
 
-const regex = /(<div.+?>)(.*)(<\/div>)/g
+const regex = /^(<div.+?>)(.*)(<\/div>)$/g
 const regexTagStart = /<\w+>/g
 const regexTagEnd = /<\/\w+>/g
 
 const HASHTAG_TYPE = 'org.symphonyoss.taxonomy.hashtag'
 const CASHTAG_TYPE = 'org.symphonyoss.fin.security.id.ticker'
 const MENTION_TYPE = 'com.symphony.user.userId'
-const EntityRegex = 'id=\"\\d+\">(.*?)<'
```

Comment:
This was not used in the code and is not documented.

Reply:
hello @symphony-elias , unfortunately, this regex still has ambiguity.